cmd/users/main.go

```diff
@@ -67,28 +67,29 @@ const (
 )
 
 type config struct {
-	LogLevel            string        `env:"SMQ_USERS_LOG_LEVEL"           envDefault:"info"`
-	AdminEmail          string        `env:"SMQ_USERS_ADMIN_EMAIL"         envDefault:"admin@example.com"`
-	AdminPassword       string        `env:"SMQ_USERS_ADMIN_PASSWORD"      envDefault:"12345678"`
-	AdminUsername       string        `env:"SMQ_USERS_ADMIN_USERNAME"      envDefault:"admin"`
-	AdminFirstName      string        `env:"SMQ_USERS_ADMIN_FIRST_NAME"    envDefault:"super"`
-	AdminLastName       string        `env:"SMQ_USERS_ADMIN_LAST_NAME"     envDefault:"admin"`
-	PassRegexText       string        `env:"SMQ_USERS_PASS_REGEX"          envDefault:"^.{8,}$"`
-	ResetURL            string        `env:"SMQ_TOKEN_RESET_ENDPOINT"      envDefault:"/reset-request"`
-	JaegerURL           url.URL       `env:"SMQ_JAEGER_URL"                envDefault:"http://localhost:4318/v1/traces"`
-	SendTelemetry       bool          `env:"SMQ_SEND_TELEMETRY"            envDefault:"true"`
-	InstanceID          string        `env:"SMQ_USERS_INSTANCE_ID"         envDefault:""`
-	ESURL               string        `env:"SMQ_ES_URL"                    envDefault:"nats://localhost:4222"`
-	TraceRatio          float64       `env:"SMQ_JAEGER_TRACE_RATIO"        envDefault:"1.0"`
-	SelfRegister        bool          `env:"SMQ_USERS_ALLOW_SELF_REGISTER" envDefault:"false"`
-	OAuthUIRedirectURL  string        `env:"SMQ_OAUTH_UI_REDIRECT_URL"     envDefault:"http://localhost:9095/domains"`
-	OAuthUIErrorURL     string        `env:"SMQ_OAUTH_UI_ERROR_URL"        envDefault:"http://localhost:9095/error"`
-	DeleteInterval      time.Duration `env:"SMQ_USERS_DELETE_INTERVAL"     envDefault:"24h"`
-	DeleteAfter         time.Duration `env:"SMQ_USERS_DELETE_AFTER"        envDefault:"720h"`
-	SpicedbHost         string        `env:"SMQ_SPICEDB_HOST"              envDefault:"localhost"`
-	SpicedbPort         string        `env:"SMQ_SPICEDB_PORT"              envDefault:"50051"`
-	SpicedbPreSharedKey string        `env:"SMQ_SPICEDB_PRE_SHARED_KEY"    envDefault:"12345678"`
-	PassRegex           *regexp.Regexp
+	LogLevel               string        `env:"SMQ_USERS_LOG_LEVEL"                 envDefault:"info"`
+	AdminEmail             string        `env:"SMQ_USERS_ADMIN_EMAIL"               envDefault:"admin@example.com"`
+	AdminPassword          string        `env:"SMQ_USERS_ADMIN_PASSWORD"            envDefault:"12345678"`
+	AdminUsername          string        `env:"SMQ_USERS_ADMIN_USERNAME"            envDefault:"admin"`
+	AdminFirstName         string        `env:"SMQ_USERS_ADMIN_FIRST_NAME"          envDefault:"super"`
+	AdminLastName          string        `env:"SMQ_USERS_ADMIN_LAST_NAME"           envDefault:"admin"`
+	PassRegexText          string        `env:"SMQ_USERS_PASS_REGEX"                envDefault:"^.{8,}$"`
+	ResetURL               string        `env:"SMQ_TOKEN_RESET_ENDPOINT"            envDefault:"/reset-request"`
```

Comment:
No need for both this and `PasswordResetURLPrefix`. Let's only use the `PasswordResetURLPrefix`.